packages/perseus/src/hints-renderer.tsx

```diff
@@ -195,10 +189,8 @@ class HintsRenderer extends React.Component<Props, State> {
                     ref={"hintRenderer" + i}
                     key={"hintRenderer" + i}
                     apiOptions={apiOptions}
-                    // @ts-expect-error [FEI-5003] - TS2339 - Property 'findExternalWidgets' does not exist on type 'Readonly<{ className?: string | undefined; hints: readonly any[]; hintsVisible?: number | undefined; }> & Readonly<{ children?: ReactNode; }>'.
```

Comment:
Typescript is now fine with this line, but ESLint's `react/prop-types` rule complains with: `'findExternalWidgets' is missing in props validation` [react/prop-types](https://github.com/jsx-eslint/eslint-plugin-react/tree/master/docs/rules/prop-types.md)

Reply:
I think we can get rid of that rule since we're using TS for prop type checking.